app/src/components/change_password.rs

```diff
@@ -72,6 +63,8 @@ pub enum Msg {
     SubmitNewPassword,
     RegistrationStartResponse(Result<Box<registration::ServerRegistrationStartResponse>>),
     RegistrationFinishResponse(Result<()>),
+    PasswordPolicyResponse(lldap_frontend_options::PasswordPolicyOptions),
+    ShowError(String),
```

Comment:
The more common pattern is 
```suggestion
    PasswordPolicyResponse(Result<lldap_frontend_options::PasswordPolicyOptions>),
```